test/clj/chronograph/domain/invite_test.clj

```diff
@@ -0,0 +1,45 @@
+(ns chronograph.domain.invite-test
+  (:require [clojure.test :refer :all]
+            [chronograph.domain.invite :as invite]
+            [chronograph.db.invite :as db-invite]
+            [chronograph.db.core :as db]
+            [chronograph.factories :as factories]
+            [chronograph.fixtures :as fixtures]
+            [chronograph.test-utils :as tu]))
+
+(use-fixtures :once fixtures/config fixtures/datasource)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest find-or-create-test
+  (testing "Given a org id and an email, it creates an invite"
+    (let [{user-id :users/id} (factories/create-user)
+          {organization-id :organizations/id} (factories/create-organization user-id)
+          invite (invite/find-or-create! db/datasource organization-id "test@email.com")]
```

Comment:
I think an alias for invite/find-or-create! would help reduce noise of db/datasource. Maybe a `(def find-or-create! (partial invite/find-or-create! db/datasource))``

Reply:
Same for invite/find-by-org-id

Reply:
I'd prefer not to add an alias here in particular because it's the subject under test.